nifi-extension-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/util/TestSFTPTransfer.java

```diff
@@ -262,7 +263,7 @@ public void testFileTypeUnknown() throws IOException {
         when(processContext.getProperty(eq(FileTransfer.REMOTE_PATH))).thenReturn(new MockPropertyValue("."));
 
         try (SFTPClient sftpClient = mock(SFTPClient.class)) {
-            when(sftpClient.ls(any(), any())).then(invocation -> {
+            when(sftpClient.ls(any(), ArgumentMatchers.<RemoteResourceFilter>any())).then(invocation -> {
```

Comment:
@EndzeitBegins is this addition required?

Reply:
Hey David. I'm afraid this change is required as [I introduced a new overload to `sshj`](https://github.com/hierynomus/sshj/pull/928) for `ls` in order to be able to `break` from listings on large directories. 
Up to `0.38.0` there were:
- `ls(String path)`, and 
- `ls(String path, RemoteResourceFilter filter)`.
With `0.39.0` the new overload `ls(String path, RemoteResourceSelector selector)` is added.

Due to this the compiler cannot infer the type for the second `any()` anymore.